src/_actions/assetMakes.actions.js

```diff
@@ -26,6 +21,6 @@ export const loadAssetMakes = () => {
  *
  * @return {object} type and payload
  */
-export const loadAssetMakesSuccess = (assetMakes) => {
-  return { type: LOAD_ASSET_MAKES_SUCCESS, payload: assetMakes };
-}
+export const loadAssetMakesSuccess = assetMakes => (
+  { type: LOAD_ASSET_MAKES_SUCCESS, payload: assetMakes }
```

Comment:
This is a comment that can be ignored for now, when using `()` as a return for arrow function, it's aesthetically pleasing to structure the contents as given in the example
```
export const loadAssetMakesSuccess = assetMakes => ({
  type: LOAD_ASSET_MAKES_SUCCESS,
  payload: assetMakes
});
```
This should be the same for all similar instances in the current changes 🙂